BD-5037: Braze Pilot docs (#10383)
braze-inc/braze-docs: _docs/_user_guide/getting_started/braze_pilot/overview.md

---
nav_title: Overview
article_title: Braze Pilot Overview
page_order: 1
page_type: reference
description: "This reference article briefly covers the integration steps required from your engineers or developers."
---

# Braze Pilot overview

> This page introduces the core concepts you’ll need to know when working with Braze Pilot.

> Braze Pilot is a mobile app that is designed to connect seamlessly with your Braze dashboard. This allows you to launch campaigns and Canvases to the app to see Braze messages come to life on your own phone. Pilot includes a library of app simulations for fictional brands representing different industries, which allows you to experience how your messaging might look from your consumers’ perspective.

Reviewer: ```suggestion
Braze Pilot is a mobile app that is designed to connect seamlessly with your Braze dashboard. This empowers you to launch campaigns and Canvases from the app, bringing Braze messages to life on your phone. Braze Pilot includes a library of app simulations for fictional brands representing different industries, allowing you to experience how your messaging might look from your customers’ perspective.
```

## Pilot app simulations

The core of Braze Pilot is its library of app simulations. Each app is a realistic simulation of an industry-specific fictional brand, instrumented to log a rich assortment of events and attributes that create endless opportunities for powering common Braze use cases.

### Steppington

> Steppington is a fitness app with workouts, exercise goals, and a Steppington+ premium service. It offers several places to demonstrate Content Cards, a section that can be revealed with Feature Flags, and a robust library of custom event logging that make it possible to illustrate many customer journeys for this industry.

Reviewer: Maybe add an example or two of places you can demonstrate Content Cards.

![The home page for Steppington with icons for marathon training, yoga, cycling, and weights.]({% image_buster /assets/img/braze_pilot/steppington_app.png %}){:style="max-width:50%"}

### PantsLabyrinth

PantsLabyrinth is an eCommerce app that sells (you guessed it) pants! The PantsLabyrinth app includes full shopping cart checkout experience, an optional wishlist feature that can be enabled with a feature flag, and many opportunities for sly jokes with friends from the UK.

![A product page for PantsLabyrinth with options to add jeans to cart.]({% image_buster /assets/img/braze_pilot/pantslabyrinth_app.png %}){:style="max-width:50%"}

### MovieCanon 

MovieCanon is a streaming service perfectly designed to illustrate common Braze use cases around content engagement. 

![The MovieCanon app with different thrillers to watch.]({% image_buster /assets/img/braze_pilot/moviecanon_app.png %}){:style="max-width:50%"}

## How Pilot connects with your Braze dashboard

> The Braze SDK is a package of code that collects data from your users once integrated with your app or website. When you connect Pilot to your dashboard, you initialize this connection between the Pilot app on your phone and the Braze SDK, as well as establish a unique connection with your Braze instance by giving Pilot your API key identifier for your dashboard.

Reviewer: ```suggestion
The Braze SDK is a code package that collects data from your users once it's integrated with your app or website. When you connect Pilot to your dashboard, you initialize this connection between the Pilot app on your phone and the Braze SDK, and establish a unique connection with your Braze instance by giving Pilot your API key identifier for your dashboard.
```
The last sentence is a little confusing. Is it saying that to connect Pilot to your dashboard, you initialize the connection between the app and the Braze SDK, and then create a unique connection with your Braze instance by giving Pilot your dashboard's API key identifier?

<div class="imgDiv">
<img src="{% image_buster /assets/img/braze_pilot/status_identified_user.png %}" style="max-width:40%">
</div>
<br>

{% endtab %}
{% tab Not connected %}

**Not connected** indicates you haven't yet initialized the Braze SDK connection with Pilot.

<div class="imgDiv">
<img src="{% image_buster /assets/img/braze_pilot/status_not_connected.png %}" style="max-width:40%">
</div>
<br>

{% endtab %}
{% endtabs %}

## Campaigns and Canvases

Campaigns and Canvases are how you send messages to your users. 

- Campaigns are best for single messages sent to a specific audience segment across various channels. 
- Canvases are advanced campaign workflows that allow you to automate and orchestrate personalized customer journeys across multiple channels. Within a Canvas, you can set up branching logic, delays, decision points, and conversion events to guide customers through a series of interactions. Canvases help ensure consistent and seamless communication across different touch points, increasing the chances of customer engagement and conversion.

## Supported messaging channels

> Braze Pilot supports the following messaging channels:

Reviewer: Minor suggestion to update this to specify that Braze Pilot "currently" supports, or to combine this and the bullet point into a sentence like "Braze Pilot currently supports in-app messages, which appear in your app to deliver timely messaging while the user is actively engaging."
